habitat-baselines/habitat_baselines/rl/hrl/hl/planner_policy.py

```diff
@@ -81,6 +87,14 @@ def _get_pred_hash(preds):
 
                 # Use set so we filter out duplicate predicates.
                 new_pred_set = list(cur_node.cur_pred_state)
+                if action.name == "nav":
+                    # Remove the at precondition, since we are walking somewhere else
+                    robot_to_nav = action._param_values[-1]
+                    new_pred_set = [
+                        pred
+                        for pred in new_pred_set
```

Comment:
`for pred in pred_set`
This will make it easier to read how `pred_set` and `new_pred_set` are related?

Reply:
Sorry, I don't understand the comment. You mean to replace all `new_pred_set` to `pred_set`?